### DISCUSSION TITLE
Layout for Login Screeen (Decoupled Mode)

### DISCUSSION THREAD

**Discussion by jaypeemayo** [OrchardCMS/OrchardCore#10335] [Q&A]
I have a decoupled OC site and I want have my own Layout for the Login screen.

As we all know, decoupled site has no themes. and even if tick 'use site theme' in admin, it will just use the default Layout.

Is there a way to set my own layout for the login screen?

**Comment by ns8482e** [comment]
Create  `Views/_ViewStart.cshtml`  and  set  Layout = `/Views/Shared/_Layout.cshtml`

**Reply by jaypeemayo** [reply]
Solve it by creating my own admin theme
